src/startupscriptgenerator/src/common/buildManifestHelper.go

```diff
@@ -7,49 +7,81 @@ package common
 
 import (
 	"common/consts"
-	"log"
+	"fmt"
+	"os"
 	"path/filepath"
 
 	"github.com/BurntSushi/toml"
 )
 
 type BuildManifest struct {
-	StartupFileName          string
-	ZipAllOutput             string
-	OperationID              string
-	VirtualEnvName           string
-	PackageDir               string
-	CompressedVirtualEnvFile string
-	StartupDllFileName       string
-}
-
-var _buildManifest = BuildManifest{}
-var _readManifest = false
+	// Flag to indicate of the manifest file exists or not
+	Exists bool
```

Comment:
this seems (IMO) like a rather non-standard way to signal existence. Why not just set the `BuildManifest` value to `nil`?

Reply:
Yeah, actually that was the first thing I tried too..i.e accessing it as a pointer and checking for `nil`, but the thing is it caused the code to be awkward impacting readability..so where ever we did `manifest.VirtualEnvironmentName` now we need to null check for `manifest` and then do it...golang does not have `manfiest?.VirtualEnvironmentName` like in C#.

Reply:
possible solution:
1. create a BuildManifest [interface](https://gobyexample.com/interfaces)
2. create a `TomlFileBuildManifest` implementation
3. create a `NilBuildManifest` implementation that just returns `nil`s or empty strings for every field

Reply:
That would involve using reflection on the interface to check whether a method received a `TomlFileBuildManifest` or `NilFileBuildManifest` to figure if a manifest file exists to not. It feels a bit heavy for that IMO. I now updated this code to create a wrapper type instead. Let me know how it looks. It simulates a Nullable type in c# (but not exactly though). In C#, we have `HasValue` and `Value` properties

Reply:
I simplified this in latest commit. Instead of checking for Exists, I just check if a value is empty or not. Take a look and let me know.